plugin/pkg/scheduler/factory/factory.go

```diff
@@ -136,19 +137,59 @@ func NewConfigFactory(
 	// We construct this here instead of in CreateFromKeys because
 	// ScheduledPodLister is something we provide to plug in functions that
 	// they may need to call.
-	var scheduledPodIndexer cache.Indexer
-	scheduledPodIndexer, c.scheduledPodPopulator = cache.NewIndexerInformer(
-		c.createAssignedNonTerminatedPodLW(),
-		&v1.Pod{},
-		0,
-		cache.ResourceEventHandlerFuncs{
-			AddFunc:    c.addPodToCache,
-			UpdateFunc: c.updatePodInCache,
-			DeleteFunc: c.deletePodFromCache,
+	podInformer.Informer().AddIndexers(cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
Oh, I see this isn't one of the generated ones now that I've read more. Can you add a comment?